### ISSUE TITLE
a question about the widerface dataset

### ISSUE THREAD

**Issue by BardeIl** [deepinsight/insightface#1199]
in the paper it said“annotate
five facial landmarks (i.e. eye centres, nose tip and mouth
corners) on faces that can be annotated from the WIDER
FACE training and validation subsets. In total, we have annotated 84.6k faces on the training set and 18.5k faces on
the validation set.”
but in the baudu cloud file zip the val label.txt just have the bbox label,so i want to ask where is the 18.5k facial landmarks label?

**Comment by freefly518** [comment]
i also ask this question
